src/executors/RExecutor.ts

```diff
@@ -0,0 +1,63 @@
+import {ChildProcessWithoutNullStreams, spawn} from "child_process";
+import {Outputter} from "src/Outputter";
+import {ExecutorSettings} from "src/settings/Settings";
+import AsyncExecutor from "./AsyncExecutor";
+import ReplExecutor from "./ReplExecutor.js";
+
+
+export default class RExecutor extends ReplExecutor {
+
+	process: ChildProcessWithoutNullStreams
+
+	constructor(settings: ExecutorSettings, file: string) {
+		//use empty array for empty string, instead of [""]
+		const args = settings.RArgs ? settings.RArgs.split(" ") : [];
+		
+		let conArgName = `notebook_connection_${Math.random().toString(16).substring(2)}`;
+
+		args.unshift(`-e`, `${conArgName}=file("stdin", "r"); while(1) { eval(parse(text= tail(readLines(con = ${conArgName}, n=1)))) }`)
+		
+		/*`			

```

Comment:
Can this be removed?

Reply:
No-- this is the code for the REPL itself. Rscript doesn't have a REPL built-in, and I didn't want to add a dependency on `R` (although the vast majority of people with Rscript have R on the path as well, I didn't want to add another path configuration-- I feel like that would confuse the user, especially when something like this is more than possible).

Reply:
Sorry, the this comment is rendered looks a bit confusing, I was referring to the commented out code below.

Reply:
I added some comments as well, and added an "/* R */" comment similar to the ones for embedded Python.